bootstrap/src/uri/mvd-uri.ts

```diff
@@ -108,8 +108,9 @@ export class MvdUri implements ZLUX.UriBroker {
     }
     const protocol = window.location.protocol;
     const wsProtocol = (protocol === 'https:') ? 'wss:' : 'ws:';
-    return `${wsProtocol}//${window.location.host}${this.pluginRootUri(plugin)}`
+    const uri = `${wsProtocol}//${window.location.host}${this.pluginRootUri(plugin)}`
         + `services/${serviceName}/${version}/${relativePath}`;
+    return proxy_mode ? uri.replace('ui', 'ws') : uri;
```

Comment:
This needs a comment, and a minor change.
It should be '/ui/' with '/ws/'. This is regex so it may need '\/' for slashes.
AND, it needs a comment above, saying why this is being done.
Essentially at this time we know it to be a hardcoded behavior that we must reflect here, so we are making an assumption that proxy_mode = api layer, and api layer's ws always is /ws, and everything else we care about is /ui.